src/openms/include/OpenMS/MATH/STATISTICS/PosteriorErrorProbabilityModel.h

```diff
@@ -125,13 +150,13 @@ namespace OpenMS
       }
 
       ///computes the gaussian density at position x with parameters params.
-      double getGauss(double x, const GaussFitter::GaussFitResult & params)
+      static double getGauss_(double x, const GaussFitter::GaussFitResult & params)
```

Comment:
could this be a free function somewhere in `Math::` ?
I guess we'd have multiple use cases in OpenMS for this (same with Gumbel)

Reply:
we will probably remove the fitting of Gauss and Gumble but I can move that to Math if it is considered useful